test/base_test.go

```diff
@@ -0,0 +1,125 @@
+package test
+
+import (
+	"context"
+	"github.com/segmentio/kafka-go"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"testing"
+	"time"
+)
+
+
+const (
+	httpUrlPrefix                   = "http://localhost:8080"
+	metricsEndpoint                 = "/metrics"
+	canaryTopicName                 = "__strimzi_canary"
+	metricServerUpdateTimeInSeconds = 30
+	kafkaServer = "localhost:9092"
```

Comment:
Quick question. This local HTTP and the kafka server will always run at
the same host where the test suite is executed from? Or are you anticipating
scenarios where test suite runs in a given machine but kafka is running 
somewhere else?

Reply:
Yes at current situation I expect they run at the same host, but in future I can imagine adding scenario with them separated even more than just by having kafka running inside container.